react-components/src/talons/Wishlist/useAddToWishlistEvent.js

```diff
@@ -37,11 +37,31 @@ const useAddToWishlistEvent = (props = {}) => {
 
     useEventListener(document, 'aem.cif.add-to-wishlist', async event => {
         const items = typeof event.detail === 'string' ? JSON.parse(event.detail) : event.detail;
-        items.forEach(item => {
-            addProductToWishlist({
-                variables: { wishlistId: '0', itemOptions: productMapper(item) }
+
+        const promises = items.map(item =>
+            addProductToWishlist({ variables: { wishlistId: '0', itemOptions: productMapper(item) } })
+        );
+
+        let toastEvent;
+        try {
+            // Wait for all items to be added to the wishlist
+            await Promise.all(promises);
+            toastEvent = new CustomEvent('aem.cif.toast', {
```

Comment:
A `toast` is something visual, however that event may be interpreted and used differently by storefront implementations.

Lets better use one that better reflects what the vent is about. I would suggest `aem.cif.add-to-wishlist.done` and 
`aem.cif.add-to-wishlist.failed`. 

We should also send the variables used before along with these events.